lutro.h

```diff
@@ -9,6 +9,13 @@
 #define PATH_MAX_LENGTH 4096
 #endif
 
+#define VERSION_MAJOR 0
+#define VERSION_MINOR 0
+#define VERSION_PATCH 1
+#define STR_(x) #x
+#define STR(x) STR_(x)
+#define VERSION_STRING "v" STR(VERSION_MAJOR) "." STR(VERSION_MINOR) "." STR(VERSION_PATCH)
```

Comment:
Rather then doing the `STR()` workaround here, it may be easier to just use `"0.0.1"`. Cannot guarantee how different compilers interpret that.